### DISCUSSION TITLE
Default Theme set in package.json is not picked up on first load.

### DISCUSSION THREAD

**Discussion by najaarv** [eclipse-theia/theia#7966] [General]
Hi, I have the below configuration in my package.json:
`"config": {
                "applicationName": "My Application",
                "defaultTheme": "one-dark-pro-monokai-darker",
                "preferences": {
                    "workbench.colorTheme": "one-dark-pro-monokai-darker"
                }
          }`
So, 'one-dark-pro-monokai-darker' is a vs code theme which I'd like to set as my default theme.
I have a theia extension in which I have registered it as part of `FrontendApplicationContribution`, like this:
`MonacoThemingService.register({
            id: "one-dark-pro-monokai-darker",
            label: "One Dark Pro Monokai Darker Theme",
            json: OneDarkProMonokaiDarkerColorTheme
});`
But, when I load my application for the first time, the dark theme, which happens to be the fallback theme is always shown, and on subsequent reloads, the One Dark Pro Monokai Darker Theme is set. What's happening and how could I resolve this? Is there any workaround or something by which I can reset the application theme after it has reached the ready state or something?

**Comment by akosyakov** [comment]
There is a known bug unfortunately: https://github.com/eclipse-theia/theia/issues/6999 Which is not easy to fix.
